<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="Contendor" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QListWidget" name="extraction_list">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>256</width>
        <height>381</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>RGB - Red Plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB - Green Plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RGB - Blue Plane</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string/>
       </property>
       <property name="background">
        <brush brushstyle="HorPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
     </widget>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>450</y>
        <width>156</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(249, 249, 249);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
      <property name="centerButtons">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="blackAndWhite">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Black &amp; White</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
